### DISCUSSION TITLE
Keyboard shortcuts for previous media and next one are much needed

### DISCUSSION THREAD

**Discussion by dagandrew** [sircharlo/meeting-media-manager#2766] [Ideas] [answered]
It was pretty fast on the old app to use them

**Comment by sircharlo** [comment]
Yeah, it's on the roadmap.

**Comment by mtdvlpr** [comment]
See #2781

**Comment by sircharlo** [comment, marked as answer]
Initial implementation in fbfaea626ba4a54549c14645a7243415c8f1e865; I look forward to your feedback!

**Comment by dagandrew** [comment]
@sircharlo Works great! thanks. I've noticed that previously Next media button switched to the next media if the main backroung was shown and if they video or image was playing, it stopped it and switched to the background. Now it only switches and does nothing when they video is playing (although it switches when the picture is shown). Can you do either create a new shortcut for stopping the video or picture or like in the previous version keep the same button for that?

**Reply by mtdvlpr** [reply]
Continued in #3035
